### PR TITLE
proxy with airgap

### PR DESCRIPTION
**Pull request by areed**
Configure proxy even when airgap. Do not make any outbound requests to test proxy in case it's an airgap install.

### CODE REVIEW

File: scripts/common/proxy.sh [hunk 1]
```diff
@@ -1,14 +1,12 @@
 
 function proxy_bootstrap() {
-    if [ "$AIRGAP" = "1" ]; then
-        return
-    fi
-
     if [ -n "$HTTP_PROXY" ]; then
         ENV_PROXY_ADDRESS="$HTTP_PROXY"
+        export https_proxy="$HTTP_PROXY"
```
Comment (areed) [inline scripts/common/proxy.sh:5]: This ensures the address can be used by curl to reach https urls

File: scripts/common/proxy.sh [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
         printf "The installer will use the proxy at '%s' (imported from env var 'HTTP_PROXY')\n" "$ENV_PROXY_ADDRESS"
     elif [ -n "$http_proxy" ]; then
         ENV_PROXY_ADDRESS="$http_proxy"
+        export https_proxy="$HTTP_PROXY"
```
Comment (salahalsaleh) [inline scripts/common/proxy.sh:9]: should this be `$http_proxy` instead?

Comment (areed) [reply scripts/common/proxy.sh:9]: It has to be https_proxy or curl ignores it for https urls

Comment (salahalsaleh) [reply scripts/common/proxy.sh:9]: i meant `$http_proxy` instead of `$HTTP_PROXY`

**Pull request merged**